galaxy_ng/app/migrations/0025_add_content_guard_to_distributions.py

```diff
@@ -0,0 +1,34 @@
+from django.db import migrations
+
+def add_content_guard(apps, schema_editor):
+    """
+    Enforce that all distributions have a content guard applied
+    """
+    AnsibleDistribution = apps.get_model('ansible', 'AnsibleDistribution')
+    ContentGuard = apps.get_model('galaxy', 'CollectionDownloadContentGuard')
+
+    cg = ContentGuard(
+        pulp_type='ansible.ansible'
+    )
+    cg.save()
+
+    AnsibleDistribution.objects.filter(
+        content_guard=None,
+        pulp_type='ansible.ansible'
+    ).update(
+        content_guard=ContentGuard.objects.first(),
```

Comment:
I suggest it to be more specific, use the `cg` object created above instead of getting the first content guard